src/scss/main.scss

```diff
@@ -755,6 +755,14 @@ tr:nth-child(odd) {
     margin: 0 10px 0 0;
 }
 
+#plainTextArea,
```

Comment:
it would be great to be able to achieve this without adding custom css ID

Reply:
The `#plainTextArea` id was already there for the txt preview and I think it is necessary to dynamically change the content of the textarea element used for the preview.
Following the same logic I added an additional div with an id for the parsed md content.
Of course this could be achieved with some css selectors but I think using an id here is how it should be done as it reduces the amount of js to find the elements.

Reply:
An element can have an id for easy js retrieval, but it's better to use only meaningful classes in css.